docs/data/destinations/braze-cohort.md

```diff
@@ -36,9 +42,17 @@ To configure an cohort integration from Amplitude to Braze, you need the followi
 5. When finished, save.
 
 ## Send a cohort
+To sync your first cohort, follow these steps:
+
+1. In Amplitude, open the cohort you want to sync, then click **Sync**.
+2. Select Braze, then click **Next**.
+3. Choose the account you want to sync to.
+4. Choose the sync cadence.
+5. When finished, save your work.
+
+## Using the Cohort in Braze
```

Comment:
nit: please add empty lines around headers to clear lint errors

Reply:
Thanks for the reminder Justin! Appreciate it.